packages/python3.vm/python3.vm.nuspec

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>python3.vm</id>
+    <version>3.10.0.20231019</version>
+    <description>Metapackage to install common Python to ensure all packages use the same Python version.</description>
+    <authors>Mandiant</authors>
+    <dependencies>
+      <dependency id="python3" version="[3.10.0, 3.11.0)" />
```

Comment:
`[3.10.0, 3.11.0)` is the ending `)` a typo or doesn't matter?

Reply:
It is on purpose to install 3.10.x, see nuspec versioning: https://learn.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges